### PR TITLE
ENG: Turkish Title Update

### PR DESCRIPTION
**Pull request by durul**
TR: Türkçe Başlık Güncellemesi

### CODE REVIEW

File: content.json [hunk 1]
```diff
@@ -1700,7 +1700,7 @@
     "title": "Turkish Language",
     "categories": [
       {
-        "title": "iOS Development Blogs",
+        "title": "iOS Geliştirici Blogları",
         "slug": "ios-development",
         "description": "No matter if it covers Swift or Objective-C, whether it's focused on app architecture or UI coding. As long as it's related to iOS development then it belongs here.",
```
Comment (daveverwer) [inline content.json]: Could this description be translated too please? Thanks!

**Comment by daveverwer** [comment]
Actually don't worry about this one as @strawb3rryx7 got the description done too. Thanks!

**Comment by durul** [comment]
ok cool....

**Comment by strawb3rryx7** [comment]
@durul What's the problem about description exactly?

**Comment by durul** [comment]
@strawb3rryx7 please check latest commit.

**Comment by strawb3rryx7** [comment]
@durul Thanks for fixing my mistake. I didn't think to translate that word. You know, some words are global in that culture, but it's looks perfect now.

**Pull request closed without merging**